webi-docker.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+VersionOfUbuntuContainer="22.04"
+
+function timestamp() {
+    echo "[+] $(date +'%F %T') [INFO] $*"
+    echo
+}
+
+function timestampEnding() {
+    echo "[+] $(date +'%F %T') [INFO] $*"
+}
+
+function err() {
+    echo "[-] $(date +'%F %T') [ERROR] $*" >&2
+    echo
+}
+
+function command_start() {
+    timestamp "Command $* has been started."
+    if ! "$@"; then
+        err "Command $* went wrong."
+        exit
+    fi
+    timestamp "Command $* has been ended."
+}
+
+function removeOldDockerContainers() {
```

Comment:
All function declarations should follow the `snake_case()` convention.
Also remove the non-Posix `function ` since we're about to remove all of those anyway.